KP-Steam.csproj

```diff
@@ -36,7 +36,7 @@
     </ItemGroup>
 
     <ItemGroup>
-      <Reference Include="Steamworks.NET, Version=12.0.0.0, Culture=neutral, PublicKeyToken=null">
+      <Reference Include="Steamworks.NET, Version=14.0.0.0, Culture=neutral, PublicKeyToken=null">
```

Comment:
@veteran29 This version of Steamworks.NET is the source from v14 with the changes I merged into v15 with rlabrecque/Steamworks.NET#440.